src/genie/libs/parser/iosxe/tests/ShowWlanIdClientStats/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,199 @@
+expected_output = {
+    "wlan_info": {"profile_name": "lizzard_Global", "id": 17},
+    "current_client_state_statistics": {
+        "authenticating": 7,
```

Comment:
I would suggest to have `wlan_id` as parent key.
```suggestion
expected_output = {
    "wlan_id": {
        17: {
            "profile_name": lizzard_Global",
            "current_client_state_statistics":
                "authenticatiing": 7,
```
this schema will be easy when we need to merge 2 parsed_output across different wlan_id to one.

Reply:
Makes sense, updated!